web/pimcore/static6/js/pimcore/settings/email/log.js

```diff
@@ -11,7 +11,7 @@
  * @license    http://www.pimcore.org/license     GPLv3 and PEL
  */
 
-pimcore.registerNS("pimcore.settings.email.log");
+pimcore.registerNS('pimcore.settings.email.log');
```

Comment:
Please avoid such formatting stuff (quote change throughout the file) as it makes review of the actual changes done in a PR way harder. Happy to accept formatting/style changes, but in a dedicated change. Anyways, merging this as it is :)

Reply:
understand!